concepts/modularization/modularization_overview.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Modularization in Kyma
+---
+
+With Kyma’s modular approach, you can install exactly the Kyma components you want, making the installation very lightweight and adjusted to your business needs. Kyma provides three components to support the modular approach: The Module Manager, the Lifecycle Manager, and the Runtime Watcher.
+
+## Kyma’s Modular Approach
+
+![](assets/modular_approach.svg)
```

Comment:
I think it is misleading to have the `Kyma CR` outside of both Control Plane and Kyma Cluster. The reason for this is that the `Kyma Instance` is in actuality the Kyma CR (they are the same). And there is an equivalent in Control Plane as well as in the Runtime.

On the other side, the `Kyma Module` has no `Config Chart`, instead the `Manifest CR` is containing image layers of which one of them is a Helm chart where another one is a configuration file. 

The Manifest is inside of the Control-Plane, and not outside like it is depicted in the graphic

Reply:
I tried implementing your suggestions with https://github.com/kyma-project/community/pull/711/commits/c8ab8c3b66743e8a681be4ee07c330beec60f900 but I'm pretty sure that's even less accurate. 
Could one of you try to adjust the image like it should be?

Reply:
😮 nice!